openr/scripts/run_openr.sh

```diff
@@ -37,7 +37,7 @@
 #
 
 # OpenR binary path or command name present on bin paths
-OPENR=openr
+OPENR=openr_bin
```

Comment:
I've wondered where this comes from - Long term lets fix what's causing this and just make it openr. With fbcode we get a binary name openr ...